src/hooks/hook_area02.nss

```diff
@@ -15,8 +15,20 @@ void main()
     // Don't run this event if the exiting object is a PC that is about to be
     // booted.
     object oPC = GetExitingObject();
-    if (GetIsPC(oPC) && GetLocalInt(oPC, LOGIN_BOOT))
-        return;
+
```

Comment:
Need a `GetIsPC()` check here.

Reply:
`GetIsPC()` added.  I originally removed it because I wasn't sure if the check would resolve correctl if a PC was logging out.  I need to do some tests on that.